.github/workflows/bump-agent-versions.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/bash
 set -e
 
+# because this script is running on the matrix, we need to fetch the current branch on every run
+current_ref=$(git rev-parse --abbrev-ref HEAD)
 package_version=$(mage integration:updatePackageVersion)
 version_requirements=$(mage integration:updateVersions)
-changes=$(git status -s -uno testing/integration/testdata/.upgrade-test-agent-versions.yml .package-version)
+stable_snapshot_version=$(mage integration:UpdateStableSnapshot $current_ref)
```

Comment:
what's the reason to capture the mage command in the variable stable_snapshot_version? I cannot find any references